docs/api-documentation/index.html

```diff
@@ -81,14 +81,14 @@ <h1 class="govuk-heading-m">Releases</h1>
   <ul class="govuk-list govuk-list--bullet">
     <li>
       <!-- UPDATE ME to upcoming latest version -->
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.4/">
-        Version 3.4 - Due for release to Sandbox on 24/06/2021. Due for release to Production on 28/06/2021.
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.5/">
+        Version 3.5 - Due for release to Sandbox on 08/07/2021. Due for release to Production on 13/07/2021.
```

Comment:
Should this be left as the 12th until the delay is agreed?

```suggestion
        Version 3.5 - Due for release to Sandbox on 08/07/2021. Due for release to Production on 12/07/2021.
```

Reply:
Had changed it to 12th but forgot to commit